cmd/backup-manager/app/backup/backup.go

```diff
@@ -193,9 +190,7 @@ func getBackupSize(backupPath string) (int64, error) {
 	if exist := util.IsFileExist(backupPath); !exist {
 		return size, fmt.Errorf("file %s does not exist or is not regular file", backupPath)
 	}
-	// Uses the same niceness level as cadvisor.fs does when running du
-	// Uses -B 1 to always scale to a blocksize of 1 byte
-	out, err := exec.Command("nice", "-n", "19", "du", "-s", "-B", "1", backupPath).CombinedOutput()
+	out, err := exec.Command("rclone", constants.RcloneConfigArg, "ls", backupPath).CombinedOutput()
```

Comment:
Maybe we could use `clone size` command? https://rclone.org/commands/rclone_size/

Reply:
Yes, if you are calculating the directory size, you can only use the `rclone size` command, but it is more convenient to use `rclone ls` when calculating the file size here, because you don't need to define the structure to deserialize the returned data.If we want to support uploading the backup data directory later, we can use `rclone size` to calculating the size  uniformly.